specification/search/data-plane/Azure.Search/preview/2021-04-30-Preview/searchservice.json

```diff
@@ -6582,6 +6668,95 @@
       "x-nullable": false,
       "description": "Represents the overall indexer status."
     },
+    "IndexerExecutionStatusDetail": {
+      "type": "string",
+      "enum": [
+        "resetDocs"
+      ],
+      "x-ms-enum": {
+        "name": "IndexerExecutionStatusDetail",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "resetDocs",
+            "name": "ResetDocs",
+            "description": "Indicates that the reset that occurred was for a call to ResetDocs."
+          }
+        ]
+      },
+      "description": "Details the status of an individual indexer execution."
+    },
+    "IndexerCurrentState": {
+      "properties": {
+        "mode": {
+          "$ref": "#/definitions/IndexingMode",
+          "readOnly": true,
+          "description": "The mode the indexer is running in."
+        },
+        "allDocsInitialChangeTrackingState": {
+          "type": "string",
+          "readOnly": true,
+          "description": "Change tracking state used when indexing all documents in the datasource starts."
```

Comment:
Should we say 'value' here as well instead of 'used'?

Reply:
I think "used" here is better since we are using different change tracking state in different places and this specific one is used in this specific place.